core/src/avm2/globals/flash/display/display_object_container.rs

```diff
@@ -141,7 +141,13 @@ pub fn get_child_at<'gc>(
     {
         let index = args.get_i32(activation, 0)?;
         return if let Some(child) = dobj.child_by_index(index as usize) {
-            Ok(child.object2())
+            // This is a hack to make AVM1Movie work- in Ruffle, Loader.content calls getChildAt.
```

Comment:
non-blocking: can you expand on this comment a little bit? I'm not entirely sure what you meant here